cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/README.md

```diff
@@ -141,4 +141,45 @@ az ad app permission add --id <service principal id> --api-permissions 1bfefb4e-
 # activate the newly added permission
 az ad app permission grant --id <service principal id> --api 00000003-0000-0000-c000-000000000000
 az ad app permission admin-consent --id <service principal id>
-```
\ No newline at end of file
+```
+
+## Profiles tool
+
+A separate tool is provided to semi-automate adding profiles in `profiles.ini`.  
```

Comment:
This should be:

"The `profiles_populator.py` tool allows semi-automatic addition of profiles to the `profiles.ini` file.

Reply:
updated